src/components/hooks/useMeeting.js

```diff
@@ -35,31 +35,34 @@ export default function useMeeting(meetingID) {
   const {meetingsAdapter} = useContext(AdapterContext);
 
   useEffect(() => {
-    // React won't recognize the meeting attributes have been updated
-    // since the state is the meeting object itself. We need to create a new
-    // meeting object trigger the state change
-    const onMeeting = (newMeeting) => {
-      setMeeting({...newMeeting});
-    };
-    const onError = (error) => {
-      setMeeting({
-        ...emptyMeeting,
-        error,
-      });
+    let cleanup;
 
-      throw error;
-    };
-    const onComplete = () => {
-      setMeeting(emptyMeeting);
-    };
+    if (!meetingID) {
+      setMeeting({...emptyMeeting});
+      cleanup = undefined;
+    } else {
+      const onMeeting = (newMeeting) => {
+        // React won't recognize the meeting attributes have been updated
+        // since the state is the meeting object itself. We need to create a new
+        // meeting object trigger the state change
+        setMeeting({...newMeeting});
+      };
+      const onError = (error) => {
+        setMeeting({...emptyMeeting, error});
+        // eslint-disable-next-line no-console
+        console.log(error);
+      };
 
-    const subscription = meetingsAdapter
-      .getMeeting(meetingID)
-      .subscribe(onMeeting, onError, onComplete);
+      const subscription = meetingsAdapter
+        .getMeeting(meetingID)
+        .subscribe(onMeeting, onError);
 
-    return () => {
-      subscription.unsubscribe();
-    };
+      cleanup = () => {
+        subscription.unsubscribe();
+      };
+    }
+
+    return cleanup;
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Let's fix this linter issue instead of disabling. We should re-run the hook if the adapter changes